src/main/scala/org/dia/algorithms/mcc/GTGRunner.scala

```diff
@@ -336,6 +336,15 @@ class GTGRunner(val masterURL: String,
     processEdges(MCCEdgeList, MCCNodeMap)
 
     /**
+     * Generate the netcdfs
+     */
+    val broadcastedNodeMap = sc.sparkContext.broadcast(MCCNodeMap)
+    val edgeTupleList = sc.sparkContext.parallelize(MCCEdgeList.toList).map({ x =>
```

Comment:
Ack'ed @rahulpalamuttam. I think it prudent to use the checkpoint option because it is unlikely that edgeListRDD computation will be required again, but the RDD will be most likely reused often. Even though we'll loose a level of fault tolerance, I'm leaning towards the localcheckPoint approach for this case as per explanation of its purpose [here](https://jaceklaskowski.gitbooks.io/mastering-apache-spark/content/spark-rdd-checkpointing.html).